viper/functions.py

```diff
@@ -113,7 +113,7 @@ def g(element, context):
                 elif isinstance(expected_arg, Optional):
                     subs.append(expected_arg.default)
                 else:
-                    raise StructureException("Not enough arguments for function %s", element)
+                    raise StructureException("Not enough arguments for function %s" % element.func.id)
```

Comment:
I think the even better fix would be: 

StructureException("Not enough arguments for function %s" % element.func.id, element)

By passing element as second parameter, the exception will report the exact the error has occured :smile: 

```
viper.exceptions.StructureException: line 3: Not enough arguments for function %s
    return as_wei_value(10)
-----------^

```